src/FootPrintsInterpolator.cpp

```diff
@@ -53,7 +54,7 @@ bool FeetInterpolator::createPhasesTimings()
     m_mergePoints.push_back(0); //attach a completely new trajectory
 
     std::shared_ptr<std::vector<StepPhase> > swing, stance;
-
+    
```

Comment:
Try to avoid trailing whitespace (https://softwareengineering.stackexchange.com/questions/121555/why-is-trailing-whitespace-a-big-deal), as this can complicate future code reviews. Most editors can be configured to automatically remove trailing whitespace.